storage_service/locations/tests/test_api.py

```diff
@@ -61,6 +61,25 @@ def test_cant_move_to_disabled_locations(self):
         # Verify error
         assert response.status_code == 404
 
+class TestPackageAPI(TestCase):
+
+    fixtures = ['base.json', 'package.json']
+    fixtures_dir = os.path.abspath(os.path.join(THIS_DIR, '..', 'fixtures', ''))
+
+    def setUp(self):
+        self.test_location = models.Location.objects.get(uuid='615103f0-0ee0-4a12-ba17-43192d1143ea')
+        # Set up locations to point to fixtures directory
+        self.test_location.relative_path = self.fixtures_dir[1:]
+        self.test_location.save()
+        ss_int = models.Location.objects.get(purpose='SS')
+        ss_int.relative_path = self.fixtures_dir[1:]
+        ss_int.save()
+
+    def tearDown(self):
+        for entry in os.listdir(self.fixtures_dir):
+            if entry.startswith('tmp'):
```

Comment:
Would it be possible for us to stub the stuff that actually creates files so we inhibit them being placed on-disk in the first place?